toolkit/phone-assessment-interviews/email-template-app-first-interview-sched.md

```diff
@@ -7,18 +7,29 @@ phase: phone-assessment-interviews
 title: Template - Email to Applicants Requesting First Interview
 ---
 
-Dear Applicant,
+Hello,
 
-Thank you for your interest in the `{POSITION-NAME}` position. You have moved forward from resume review. You're now invited to a phone interview with a subject matter expert to further assess your qualifications. This interview will last 30-60 minutes. If you move forward from this interview, you will have one additional subject matter expert qualifications interview. If you're referred for selection, you may be asked to participate in additional interviews with hiring managers.
+Thank you for your interest in the `{POSITION-NAME}` position. You have moved forward from the {PRIOR PHASE}. You’re now invited to a phone interview with a subject matter expert to further assess your qualifications. This interview will take about {TIME OF INTERVIEW} to complete. 
 
-Here are some key details about the position you've applied for:
+{IF THIS IS ONE OF TWO INTERVIEWS: If you move forward from this interview, you will have one additional subject matter expert qualifications interview.}
+{IF ONLY DOING ONE INTERVIEW: If you’re referred for selection, you may be asked to participate in additional interviews with hiring managers.}
 
-- Your job location would be in `{CITY, STATE}`. There are `{no}` reimbursements for relocation expenses.
+Please also note: the interview will be structured, meaning that questions have been determined in advance. In order to ensure fairness, you will be asked the same questions as all other applicants. Your interviewer will not be able to answer questions about the hiring process or the job you're applying for. If you have questions about the process, please contact me. You will have time to ask questions about the role after you have qualified for the position.
+
+Please refer to the job announcement for details about the position you’ve applied for: {link to job announcement}
+
+{OPTIONAL INFO TO INCLUDE}
+- Your job location would be in `{CITY, STATE}`. There are `{no}` reimbursements for relocation expenses. The position `{is/is not}` not remote-friendly.
 - This is a `{SERIES AND GRADE}` role. With `{CITY, STATE}` locality pay, the salary ranges from `{$X to $Y}` per year.
 - `{DELETE IF NOT APPLICABLE: While this position does not require a security clearance, you will undergo a background check.}`
+{/OPTIONAL INFO TO INCLUDE}
+
+{FOR SELF-SERVICE SCHEDULING}: If you’re still interested in this position, go to {LINK} to select a time you're available for a phone interview. If we don’t receive your availability by {DATE}, we’ll assume you are no longer interested and remove you from consideration. (We don't want this to happen, but we need to give you notice to ensure fairness).
```

Comment:
Can we say something friendly before "if you're stilll interested..." in both these cases? maybe "We are eager to invite you to a qualifying interview so we can learn more about your qualifications for this role. If you are still interested..."